diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -145,7 +146,7 @@ controller_interface::return_type DiffDriveController::update()
     return controller_interface::return_type::OK;
   }
 
-  const auto current_time = node_->get_clock()->now();
+  const auto current_time = time;
```

Comment:
In line 159 on e should use `period`

Reply:
that's not reeeeally `dt`, I'll rename the variable